test/duplicates.e2e-spec.ts

```diff
@@ -0,0 +1,933 @@
+import { Test } from '@nestjs/testing';
+
+// Global Helpers
+import { ethers } from 'ethers';
+import { fromHexString, toHexString } from '@chainsafe/ssz';
```

Comment:
Can you tell me if anything has changed here regarding the develop branch? Can I read it separately?

Reply:
unfortunately it is impossible to read it separately